tx-query/app/src/test/mod.rs

```diff
@@ -95,24 +95,26 @@ pub fn test_integration() {
 
         let listener = TcpListener::bind("0.0.0.0:3443").expect("failed to bind the TCP socket");
 
-        match listener.accept() {
-            Ok((stream, addr)) => {
-                info!("new client: {:?}", addr);
-                let _ = stream.set_read_timeout(Some(time::Duration::new(TIMEOUT_SEC, 0)));
-                let _ = stream.set_write_timeout(Some(time::Duration::new(TIMEOUT_SEC, 0)));
-                let mut retval = sgx_status_t::SGX_SUCCESS;
-                let result =
-                    unsafe { run_server(enclave.geteid(), &mut retval, stream.as_raw_fd()) };
-                match result {
-                    sgx_status_t::SGX_SUCCESS => {
-                        info!("client query finished");
-                    }
-                    e => {
-                        error!("client query failed: {}", e);
+        for _ in 0..2 {
```

Comment:
Is this intentional? Do you only want to listen two requests?
You can use `while let` if not.

Reply:
Yes, it's intentional in this test -- only to listen to/process two requests and then terminate.